Users/Glob_Patterns.rst

```diff
@@ -174,14 +209,10 @@ brackets. Parentheses that have no match are ignored as well as
 
```

Comment:
line 219 it is ``\*\*`` whereas it should be ``**`` (The subheading). And may other occurrences are there. As this is not related to the issue, it is up to you if you want to do it.